documentation/design-docs/ipc-protocol.md

```diff
@@ -873,28 +873,36 @@ For example, if the Diagnostic Server finds incorrectly encoded data while parsi
 
 where:
 
 * `<port address>` is a NamedPipe name without `\\.\pipe\` on Windows, and the full path to a Unix domain socket on other platforms
 * `tag ::= <SUSPEND_MODE> | <PORT_TYPE>`
-* `<SUSPEND_MODE> ::= suspend | nosuspend` (default value is nosuspend)`
+* `<SUSPEND_MODE> ::= suspend | nosuspend` (default value is suspend)`
 * `<PORT_TYPE> ::= connect` (future types such as additional listen ports could be added to this list)
 
-Any diagnostic ports specified in this configuration will be created in addition to the default port (`dotnet-diagnostic-<pid>-<epoch>`). The suspend mode of the default port is set via the new environment variable `DOTNET_DefaultDotnetPortSuspend` which defaults to 0 for `nosuspend`.
+Example usage:
+
+```shell
+$ export DOTNET_DiagnosticPorts=$DOTNET_DiagnosticPorts;~/mydiagport.sock,nosuspend;
+```
 
-Each port configuration specifies whether it is a `suspend` or `nosuspend` port. Ports specifying `suspend` in their configuration will cause the runtime to pause early on the startup path before most systems are available. This allows any agent to receive a connection and properly setup before the application startup continues. Since multiple ports can individually request suspension, the `resume` command now needs to be sent by each suspend port connection before the event will be set and the runtime resumes execution.
+Any diagnostic ports specified in this configuration will be created in addition to the default port (`dotnet-diagnostic-<pid>-<epoch>`). The suspend mode of the default port is set via the new environment variable `DOTNET_DefaultDotnetPortSuspend` which defaults to `0` for `nosuspend`.
+
+Each port configuration specifies whether it is a `suspend` or `nosuspend` port. Ports specifying `suspend` in their configuration will cause the runtime to pause early on in the startup path before most runtime subsystems have started. This allows any agent to receive a connection and properly setup before the application startup continues. Since multiple ports can individually request suspension, the `resume` command needs to be sent by each suspended port connection before the runtime resumes execution.
 
 If a config specifies multiple tag values from a tag type, for example  `"<path>,nosuspend,suspend,suspend,"`, only the first one is respected.
 
-The port address value is required for a port configuration. If a configuration doesn't specify an address and only specifies tags, then the first tag will be treated as the path. For example, a configuration `suspend,connect` would cause a port with the name `suspend` to be created, in the default `nosuspend` mode.
+The port address value is **required* for a port configuration. If a configuration doesn't specify an address and only specifies tags, then the first tag will be treated as the path. For example, a configuration `suspend,connect` would cause a port with the name `suspend` to be created, in the default `suspend` mode.
```

Comment:
```suggestion
The port address value is **required** for a port configuration. If a configuration doesn't specify an address and only specifies tags, then the first tag will be treated as the path. For example, a configuration `suspend,connect` would cause a port with the name `suspend` to be created, in the default `suspend` mode.
```

Reply:
While you're changing this, it might be better if you changed the tag being used as the example to `nosuspend,connect` and say this creates a `nosuspend` path in `suspend` mode since that will make more obvious to the reader that the first tag is being treated as a path.